### ISSUE TITLE
What is the meaning of base_lrs?

### ISSUE THREAD

**Issue by rosefun** [ildoonet/pytorch-gradual-warmup-lr#16]
I don't find the initialization of `base_lrs`? Does it initialize with 0?

**Comment by gauenk** [comment]
This is defined in the base class _LRScheduler (c.f. [see here](https://github.com/pytorch/pytorch/blob/30fccc53a9a1ae88db39ef7dd4de35bd2c174ca4/torch/optim/lr_scheduler.py#L43)). The base_lrs by default takes the "lr" parameter set by the optimizer (c.f. [see here](https://github.com/pytorch/pytorch/blob/30fccc53a9a1ae88db39ef7dd4de35bd2c174ca4/torch/optim/optimizer.py#L235-L240)). I believe the "lr" is usually contained in the "defaults" field of the optimizer, so it is not explicitly set. 

So you can initialize it to whatever when you create your optimizer.
